CONTRIBUTING.md

```diff
@@ -44,4 +44,11 @@ There are a few layers of tests - integration, unit and build tests.
 npm run test-build
 npm run test-integration
 npm run test-unit
-```
\ No newline at end of file
+```
+
+# Publish style-spec NPM package
```

Comment:
```suggestion
# Publish style-spec npm package
```

Reply:
NPM stands for Node Package Manager, it should be with capitals I believe...